components/datetime/src/time_zone.rs

```diff
@@ -110,6 +109,8 @@ pub(super) struct TimeZoneFormat<'data> {
     /// The specific short metazone names, e.g. Pacific Daylight Time
     pub(super) mz_specific_short:
         Option<DataPayload<'data, provider::time_zones::MetaZoneSpecificNamesShortV1Marker>>,
+    /// The config of TimeZoneFormat
+    pub(super) config: Option<TimeZoneFormatConfig>,
```

Comment:
> nitpick

I think it would make more sense to readers to move `config` to the top where `patterns` is, since those are the two members for initializing the TZF, and then the rest of the members are data payloads.